documentation/asciidoc/topics/attributes/community-attributes.adoc

```diff
@@ -6,6 +6,13 @@
 
 :github_root: https://github.com/infinispan/infinispan-operator/
 :github_raw: https://raw.githubusercontent.com/infinispan/infinispan-operator/
+:github_tag: 0.3.0
+
+//Environment variables
```

Comment:
These environment variables should not be in the documentation. They're internal operator details that the user does not care about and should not fiddle with.